examples/ansible/testvcertplaybook.yaml

```diff
@@ -0,0 +1,20 @@
+config:
+  connection:
+    platform: vaas
+    credentials:
+      tokenURL: <Your Service Account Token URL - Copy from VCP> 
+      externalJWT: '{{ Env "TLSPC_EXTERNAL_JWT" }}'
+certificateTasks:
+  - name: myCertificate
+    renewBefore: 31d
+    request:
+      csr: service
+      subject:
+        commonName: "ansibletest.lab.corp.net"

```

Comment:
We probably would want to use the variables here to determine the CN based on hostname?

Reply:
Agreed on the DNS SAN. I'll add that. 

I was not putting in any variables or expressions to determine the CN by the hostname as the usecase this was made for was not server certs but applications hosted by the server, which have differing FQDNs they are accessible by than the server hostname. Along the lines of  `myapp.corp.net` instead of `WKS115PKM.corp.net`

Reply:
Updated in bf95fa38a9b528b083446a14790d1543abd39f06